src/types/line.js

```diff
@@ -141,11 +143,7 @@ function calculateAutoRotation(line) {
 function drawLabel(ctx, line) {
 	const label = line.options.label;
 
-	ctx.font = fontString(
-		label.font.size,
-		label.font.style,
-		label.font.family
-	);
+	ctx.font = toFontString(label.font);
```

Comment:
Lets keep this one, although it does not really reflect anything in the PR title.

For future, try to keep a PR small and only do the thing title says. Its much easier to review that way. 
Also when you can, don't put them on top of each other (I know this is because I asked you to split them, but you could have done it differently), it also makes it easier to review and avoids the rebase after every merge.

Sorry for nagging.

Reply:
@kurkle you're right and apologize for that. Sometimes it's difficult to submit more than 1 PR without any dependencies among that. In the future I'm gonna submit 1 PR at once in order to do not have this cascade.
Again, sorry...